internal/proxy/proxy.go

```diff
@@ -328,35 +358,35 @@ func (p *Proxy) setIdledCookie(w http.ResponseWriter, pci ProxyCacheItem) {
 	return
 }
 
-func (p *Proxy) handleGitHubRequest(w http.ResponseWriter, r *http.Request) (ns string, noProxy bool) {
+func (p *Proxy) handleGitHubRequest(w http.ResponseWriter, r *http.Request, requestLogEntry *log.Entry) (ns string, noProxy bool) {
 	noProxy = true
-	//Load request body if it's GH webhok
+	//Load request body if it's GH webhook
 	gh := GHHookStruct{}
 	defer r.Body.Close()
+
 	body, err := ioutil.ReadAll(r.Body)
 	if err != nil {
-		p.HandleError(w, err)
+		p.HandleError(w, err, requestLogEntry)
 		return
 	}
+
 	err = json.Unmarshal(body, &gh)
 	if err != nil {
-		p.HandleError(w, err)
+		p.HandleError(w, err, requestLogEntry)
 		return
 	}
 
+	requestLogEntry.WithField("json", gh).Info("Processing GitHub JSON payload")
```

Comment:
I'd do this as `Debug` - probably no need to log all request payloads?

Reply:
Fair enough. I was undecided on this one. Let's do debug then.